src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Windows_NT.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -11,13 +11,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <NativeBinary Include="$(BinDir)clretwrc.dll" />
-    <NativeBinary Include="$(BinDir)coreclr.dll" />
+    <NativeBinary Include="$(BinDir)clretwrc.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)coreclr.dll" LinkerUsage="runtime" />
     <NativeBinary Include="$(BinDir)dbgshim.dll" />
-    <NativeBinary Include="$(BinDir)mscordaccore.dll" />
-    <NativeBinary Include="$(BinDir)mscordbi.dll" />
-    <NativeBinary Include="$(BinDir)mscorrc.debug.dll" />
-    <NativeBinary Include="$(BinDir)mscorrc.dll" />
+    <NativeBinary Include="$(BinDir)mscordaccore.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)mscordbi.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)mscorrc.debug.dll" LinkerUsage="diagnostic" />
```

Comment:
Isn't mscorrc used for resource strings?

Reply:
Not sure what mscorrc.dll is used for, but it isn't diagnostic related.  I also don't know what the difference between it and mscorrc.debug.dll.

Reply:
Both mscorrc.dll and mscorrc.debug.dll contain localized strings for the unmanaged runtime. We should include both by default, I think.